src/clj/puppetlabs/services/request_handler/request_handler_core.clj

```diff
@@ -2,7 +2,8 @@
   (:import (java.security.cert X509Certificate)
            (java.util HashMap)
            (java.io StringReader)
-           (com.puppetlabs.puppetserver JRubyPuppetResponse))
+           (com.puppetlabs.puppetserver JRubyPuppetResponse)
+           (org.apache.commons.io IOUtils))
```

Comment:
Good call.  I see a few other unused ones in that namespace and will remove those too.